src/Accelerators/Accelerator.hpp

```diff
@@ -42,6 +42,15 @@
 #define ACCEL_CL_ENUM_TO_STRING(name, map)                                     \
   map[accel::Accelerator::Kind::name] = #name;
 
+#define ACCEL_INSTRUMENTSTAGE_ENUM(name)                                       \
+  , nnpaAfterOnnxToOnnx, nnpaAfterOnnxToZhigh, nnpaAfterZhighToZlow
+
+#define ACCEL_INSTRUMENTSTAGE_CL_ENUM(name)                                    \
+  , clEnumVal(nnpaAfterOnnxToOnnx, #name " Profile for onnx ops."),            \
+      clEnumVal(                                                               \
+          nnpaAfterOnnxToZhigh, #name " Profile for onnx and zhigh ops."),     \
+      clEnumVal(nnpaAfterZhighToZlow, #name " Profile for zlow ops.")
+
```

Comment:
I moved NNPA related option (stage name) to Accelerator files using `APPLY_TO_ACCELERATORS()`, but I'm not sure how I can move them to NNPA files (NNPACompilerUtils.cpp etc.)...

Reply:
Moved NNPA stage name to `NNPACompilerUtils.hpp` using `INSTRUMENTSTAGE_EUM_NNPA` and `INSTRUMENTSTAGE_CL_ENUM_NNPA` by commit https://github.com/onnx/onnx-mlir/pull/1823/commits/4ff1354556d8b19981e318286b58d4562edc9492

Reply:
@chentong319 I removed stages from instrumentation pass as you suggested. I think this is better because developers can move the pass up and down easily for their needs.